src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+    <title>Register</title>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <style>
+        label{
+            display: inline-block;
+            width 100px;
+        }
+    </style>
+
+</head>
+
+<body>
+
+    <nav>
+        <a id="navTitle" href="/">CodeU Chat App </a>
+        <a href="/conversations">Conversations</a>
+
+        <% if(request.getSession().getAttribute("user") != null){
+            %>
+
+            <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+
+            <$ }else{ %>
+                <a href="/login">Login</a>
+            <% } %>
+    </nav>
+
+    <div id="container">
+
+        <h1>Register</h1>
+
+        <% if(request.getAttribute("error") != null){ %>
```

Comment:
style nit: spaces around (): "if(request...){" -> "if (request...) {"

We haven't adopted a strict style guide, so this is just a suggestion, but it's fairly standard.